etc/patches/0001-remove-the-glusterfs_repo-var-from-conf.patch

```diff
@@ -0,0 +1,25 @@
+From 04a6f332205cb937f0ca89159176a3ff5c34fcee Mon Sep 17 00:00:00 2001
```

Comment:
In this .patch file actually we are removing the entry from conf file. Actually defaulting happens in python-gdeploy and create_cluster flow :)

Dont see anything wrong in name. BUT yes `whats in name ;)`